library/psa_crypto_ffdh.h

```diff
@@ -112,4 +112,33 @@ psa_status_t mbedtls_psa_ffdh_generate_key(
     size_t key_buffer_size,
     size_t *key_buffer_length);
 
+/**
```

Comment:
Writing a note here because I cannot do it in the proper line since it was modified in a preceding PR. I just realized that perhaps the naming `mbedtls_psa_export_ffdh_public_key()` is not correct as other functions are `mbedtls_psa_ffdh_export_public_key()`. It's not something we have to fix here, but since there are PRs related to the FFDH management, it's something that can be fixed there perhaps.